### ISSUE TITLE
After Successful steps Upgrage report on CMD line, app.config still shows old 4.8 TFM

### ISSUE THREAD

**Issue by fasteddys** [dotnet/upgrade-assistant#1010]
The upgrade shows succeeded, but when you open the project file (not backup), it still shows 4.8 old framework

**Comment by sunandabalu** [comment]
@fasteddys  Looks like the upgrade was not actually successful if the TFM you are seeing is still .NET 4.8, could you please attach the `upgrade-assistant.clef` file so we can see if there are any errors that you hit during upgrade?

**Comment by fasteddys** [comment]
The tool is still very buggy, I watched the Video and followed the steps.

Also do you have an un attended full auto upgrade mode/ CLI switch?

**Comment by sunandabalu** [comment]
> Also do you have an un attended full auto upgrade mode/ CLI switch?

Yes, the option `--non-interactive` will run the tool unattended.

**Comment by Reet-Kaur** [comment]
Hi, I noticed the same issue. Is there any alternative to fix the App.config until this bug gets fixed? 

![image](https://user-images.githubusercontent.com/40193051/150193218-fbe5470b-56a9-4dd2-bf21-add8f41d10f8.png)

The tool clearly says that app config upgrade is completed. However, the app.config still contains the old values/settings
<?xml version="1.0" encoding="utf-8"?>
<configuration>
<startup><supportedRuntime version="v4.0" **sku=".NETFramework,Version=v4.8**"/></startup></configuration>

Thanks!

**Comment by sunandabalu** [comment]
Apologize I misunderstood the initial statement, I assumed the TFM is not updated in the project file but you are talking about the `app.config` file. upgrade-assistant only takes the settings from app.config , web.config and creates the appsettings.json - https://github.com/dotnet/upgrade-assistant/blob/main/src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Configuration/Updaters/AppSettingsConfigUpdater.cs - but does not edit the settings in the existing app.config file. 
@mjrousos to add some more context here.

**Comment by mjrousos** [comment]
Like @sunandabalu said, the only updates the tool makes in app.config/web.config is to migrate app settings and connection strings, and to disable config sections that won't work on .NET 6.

It's a good idea to add an additional step to clean up `<supportedRuntime>` elements since they're no longer needed and just clutter the file, but it's probably a low-priority work item since I would expect those elements to be ignored by .NET 6 and effectively be no-ops.

**Comment by sunandabalu** [comment]
@mjrousos you are right, .NET 6 does ignore app.config as a whole.

@fasteddys Feel free to put up a PR with changes recommended by @mjrousos above, we will review it and get it merged if approved.
